bokulich-lab/q2-fondue
# ENH: Scraping DOI from library

This PR allows for retrieving DOI/ISBN information when scraping accession IDs from a web Zotero library with the action `scrape_collection`. This returns an artifact of type `NCBIAccessionIDs` that newly not only includes accession IDs but also a column with the DOI information. The retrieved DOI information is preserved as an additional metadata column when running `get_metadata` (and `get_all`) on the obtained `NCBIAccessionIDs`.

This PR
* closes #107
* could be a good alternative to #102, if retrieval of DOIs from PubMed directly is not an option

**Testing:**
1. Run the command `scrape_collection` on an existing Zotero collection. Check that the obtained artifact includes DOI information of the scraped accession IDs.
2. Use a non-empty output artifact from 1. and run `get-metadata`. Check that the newly retrieved metadata includes the DOI information you had obtained from 1.
